### ISSUE TITLE
NoClassDefFoundError Checkout Listeners

### ISSUE THREAD

**Issue by percula** [serso/android-checkout#83]
I ran into this error during automated device testing. Not sure what to make of it:

```
FATAL EXCEPTION: main
 java.lang.NoClassDefFoundError: org.solovyev.android.checkout.Checkout$Listeners
 	at org.solovyev.android.checkout.Checkout.<init>(Checkout.java:110)
 	at org.solovyev.android.checkout.ActivityCheckout.<init>(ActivityCheckout.java:77)
 	at org.solovyev.android.checkout.Checkout.forActivity(Checkout.java:128)
 	at com.myapp.ui.MainActivity.checkIAB(MainActivity.java:193)
```

Where CheckIAB in my MainActivity (line 193 that causes error noted below):
```
private void checkIAB() {
        final Billing billing = MyApplication.get(this).getBilling();
        mCheckout = Checkout.forActivity(this, billing);      // Line 193
        mCheckout.start();
        mCheckout.createPurchaseFlow(new PurchaseListener());

        mSkus = new ArrayList<>();
        mSkus.add(SUPPORTER_SKU);

        refreshInventory();
    }
```

**Comment by serso** [comment]
@percula Checkout.Listeners exists in the [aar](https://oss.sonatype.org/content/repositories/releases/org/solovyev/android/checkout/1.0.0/). Try to clean/rebuild the project (possibly redownload the library). Another possible reason why it's not available at runtime is Proguard. Please check your Proguard rules.

**Comment by percula** [comment]
Thanks @serso . I have a feeling it is Proguard. I tweaked my proguard-rules.pro and will see if the error re-occurs.

**Comment by percula** [comment]
Ok, I figured it out. It was nothing to do with Checkout, just that Checkout was the first thing used since it extends the Application class. I had multidex enabled, and the app was only crashing on API 19 and less (which are the versions that don't support multidex). So any class over the 64k limit didn't exist, and the first  missing class Android encountered happened to be Checkout.Listeners. To solve this, I enabled ProGuard and disabled multidex. I apparently didn't have ProGuard enabled before, since I thought it might cause issues. The irony =)
